### PR TITLE
UI: Simplify custom Sidebar styling by adding classNames to DOM elements

### PR DESCRIPTION
**Pull request by tooppaaa**
Issue:
It became difficult to add style to the sidebar

## What I did

add classnames back

## How to test

- Inspect DOM

### CODE REVIEW

File: lib/ui/src/components/sidebar/Search.tsx [hunk 1]
```diff
@@ -354,7 +354,10 @@ export const Search = React.memo<{
           return (
             <>
               <ScreenReaderLabel {...getLabelProps()}>Search for components</ScreenReaderLabel>
-              <SearchField {...getRootProps({ refKey: '' }, { suppressRefError: true })}>
+              <SearchField
+                {...getRootProps({ refKey: '' }, { suppressRefError: true })}
+                className="search-field"
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/Search.tsx:359]: Previously accessed using `nav.container.sidebar-container form`

File: lib/ui/src/components/sidebar/SearchResults.tsx [hunk 2]
```diff
@@ -124,7 +124,7 @@ const Result: FunctionComponent<
   const nameMatch = matches.find((match: Match) => match.key === 'name');
   const pathMatches = matches.filter((match: Match) => match.key === 'path');
   const label = (
-    <div>
+    <div className="search-result-item--label">
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/SearchResults.tsx:127]: invented

File: lib/ui/src/components/sidebar/SearchResults.tsx [hunk 3]
```diff
@@ -202,7 +202,7 @@ export const SearchResults: FunctionComponent<{
       <ResultsList {...getMenuProps()}>
         {results.length > 0 && !query && (
           <li>
-            <RootNode>Recently opened</RootNode>
+            <RootNode className="search-result-recentlyOpened">Recently opened</RootNode>
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/SearchResults.tsx:205]: invented

File: lib/ui/src/components/sidebar/Tree.tsx [hunk 4]
```diff
@@ -77,6 +77,7 @@ const Node = React.memo<NodeProps>(
         <LeafNode
           key={id}
           id={id}
+          className="sidebar-item"
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/Tree.tsx:80]: already present in 6.0

File: lib/ui/src/components/sidebar/Tree.tsx [hunk 5]
```diff
@@ -97,10 +98,18 @@ const Node = React.memo<NodeProps>(
 
     if (isRoot(item)) {
       return (
-        <RootNode key={id} id={id} data-ref-id={refId} data-item-id={item.id} data-nodetype="root">
+        <RootNode
+          key={id}
+          id={id}
+          className="sidebar-subheading"
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/Tree.tsx:104]: already present in 6.0

File: lib/ui/src/components/sidebar/Tree.tsx [hunk 6]
```diff
@@ -97,10 +98,18 @@ const Node = React.memo<NodeProps>(
...
+          data-ref-id={refId}
+          data-item-id={item.id}
+          data-nodetype="root"
+        >
           {item.name}
           <Action
             type="button"
+            className="sidebar-subheading-action"
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/Tree.tsx:112]: invented

File: lib/ui/src/components/sidebar/Tree.tsx [hunk 7]
```diff
@@ -119,6 +128,7 @@ const Node = React.memo<NodeProps>(
       <BranchNode
         key={id}
         id={id}
+        className="sidebar-item"
```
Comment (tooppaaa) [inline lib/ui/src/components/sidebar/Tree.tsx:131]: already present in 6.0 (they where all sidebar-item before)

**Review by ghengeveld — APPROVED** [review]

**Review by shilman — APPROVED** [review]
LGTM!

**Pull request merged**